hibernate-reactive-core/src/main/java/org/hibernate/reactive/event/impl/AbstractReactiveSaveEventListener.java

```diff
@@ -420,7 +420,7 @@ private CompletionStage<AbstractEntityInsertAction> addInsertAction(
 			boolean useIdentityColumn,
 			EventSource source,
 			boolean shouldDelayIdentityInserts) {
-		final ReactiveActionQueue actionQueue = source.unwrap( ReactiveSession.class ).getReactiveActionQueue();
+		final ReactiveActionQueue actionQueue = source.unwrap(ReactiveSession.class).getReactiveActionQueue();
```

Comment:
Not critical, but are you using the right formatter? Doesn't the ORM formatter keep the spaces in this case?

Reply:
comparing the code with main I noticed there were no spaces so I thought it was right and removed it, going to add them back